app/Http/Controllers/ReloadlyController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Services\ReloadlyService;
+use Illuminate\Support\Facades\Log;
+
+class ReloadlyController extends Controller
+{
+    public function getBalance() {
+		Log::info("[ReloadlyService][checkAccountBalance]\t Checking Account Balance");
```

Comment:
Make sure the logging names match what you are working on. In this case it should be [ReloadlyController][getBalance]